app/views/layouts/_nav_bar.html.erb

```diff
@@ -0,0 +1,47 @@
+<nav class="navbar" role="navigation" aria-label="main navigation">
+<div class="navbar-brand">
+    <a class="navbar-item" href="/">
+    <img src="<%= asset_path('logo.png') %>">
+    </a>
+</div>
+<div>
+    <a role="button" class="navbar-burger burger" id='burger' aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+    <span aria-hidden="true"></span>
+    <span aria-hidden="true"></span>
+    <span aria-hidden="true"></span>
+    </a>
+</div>
+<div class="navbar-menu" id='nav-links'>
+    <div class="navbar-start">
+    <a class="navbar-item" href="/posts"> Home </a>
+    <a class="navbar-item" href="/pages/about"> About </a>
+        <%# Drop down menu for nav bar %>
+        <%# <div class="navbar-item has-dropdown is-hoverable">
```

Comment:
Would we like to keep these commented out section?

Reply:
You're right :D Thanks Mas, all updated now.